### PR TITLE
Highlight new column name

### PR DESCRIPTION
**Pull request by Jesus89**
Related to: #12662

When a new column is added its input is focused for renaming the column

highlight-column

### CODE REVIEW

**Comment by nobuti** [comment]
👍

**Comment by nobuti** [comment]
It's in ded15, in case you want to take a look @piensaenpixel.

**Comment by nobuti** [comment]
retest please

**Comment by nobuti** [comment]
retest this please

File: lib/assets/core/javascripts/cartodb3/components/table/head/table-head-item-view.js [hunk 1]
```diff
@@ -100,9 +101,9 @@ module.exports = CoreView.extend({
       val: 'order',
       isOrderBy: this._tableViewModel.get('order_by') === columnName,
       sortBy: this._tableViewModel.get('sort_order'),
-      action: function (mdl) {
+      action: function (model) {
         self._tableViewModel.set({
-          sort_order: mdl.get('sort'),
+          sort_order: model.get('sort'),
```
Comment (elenatorro) [inline lib/assets/core/javascripts/cartodb3/components/table/head/table-head-item-view.js]: Beautiful 😍

**Comment by piensaenpixel** [comment]
<3

**Pull request merged**